CHANGES.txt

```diff
@@ -37,6 +37,7 @@ Features
 
 Fixes
 
+* Fix loading resource images from a module that is not a package (#836)
```

Comment:
I think the one change this made compared to 7.1.0 was to do with fixing the behaviour of unexpectedly getting unittest `__main__` module when the code is invoked as a script from within a test 🤯 . Prior to switching from pkg_resources to importlib_resources, loading resource images from a non-package does work owing to the use of `__file__`. So it might not need to be included in the changelog?

Reply:
ah yes, that makes sense.  I will remove it, thanks!